MHVideoPhotoGallery/MMHVideoPhotoGallery/MHGalleryImageViewerViewController/MHGalleryImageViewerViewController.m

```diff
@@ -1388,10 +1393,15 @@ -(void)moviePlayBackDidFinish:(NSNotification *)notification{
     [self.view bringSubviewToFront:self.playButton];
     [self stopTimer];
     
-    self.moviePlayer.currentPlaybackTime =0;
+    [self.moviePlayer.player.currentItem seekToTime: kCMTimeZero];
     [self movieTimerChanged:nil];
     [self updateTimerLabels];
     
+}
+
+-(void)newLogError:(NSNotification *)notification{
```

Comment:
Is this still necessary?

Reply:
No, I was testing something and forgot to remove it.